packages/app/background-jobs-common/src/background-job-processor/monitoring/BackgroundJobProcessorMonitor.ts

```diff
@@ -89,19 +90,33 @@ export class BackgroundJobProcessorMonitor<
   public jobStart(job: JobType, requestContext: RequestContext): void {
     const transactionName = `bg_job:${this.ownerName}:${this.queueId}`
     this.transactionObservabilityManager.start(transactionName, resolveJobId(job))
-    requestContext.logger.info({ origin: this.processorName }, `Started job ${job.name}`)
+    requestContext.logger.info(this.buildLogParams(job), `Started job ${job.name}`)
+  }
+
+  public jobTryError(job: JobType, error: unknown, requestContext: RequestContext): void {
```

Comment:
name is slightly confusing, what does `jobTryError` mean?

Reply:
it is that the job throws an error on the process try, but it does not have to be a definitive error (the job can be retried depending on the config).

I agree that it is confusing, but I am super bad at naming, let me give it a thought but please if you have something in mind I am more than open to ideas 🙏

Reply:
Do you mean BullMQ "job attempt" or something else?

Reply:
Yes, the job attempt, maybe `jobAttemtError` is meaningful enough?

Reply:
sounds good!